Makefile

```diff
@@ -42,13 +43,14 @@ lint:
 .PHONY: lint
 
 test:
-	gotestsum --max-fails=50 ./...
+	gotestsum --max-fails=50 ./... -run "$(TESTS)"
```

Comment:
This is the change that makes sure only the specified tests are passed. We didn't need this previously because we always ran all tests.